apps/gaia/src/components/Navbar/Navbar.tsx

```diff
@@ -1,15 +1,42 @@
-import React from "react"
+import React, { FC, useState } from "react"
+import { useRouter } from "next/router"
 
-const Navbar = () => {
+interface Props {
+  childId: string
+}
+const Navbar: FC<Props> = ({ childId }) => {
+  const [active, setActive] = useState("lesson-plan")
+  const router = useRouter()
+  if (router.pathname !== "/") {
+    setActive(router.pathname.slice(1, router.pathname.length))
+  }
   return (
     <div className="sticky top-0 bg-surface border-b">
       <nav className="w-full flex max-w-3xl mx-auto pt-3">
-        <div
-          className="mx-3 py-1 border-b-2 border-black text-sm px-2"
-          style={{ marginBottom: "-1px" }}
-        >
-          Lesson Plans
-        </div>
+        <ul className="flex">
+          <li className="-mb-px mr-1" onClick={() => setActive("lesson-plan")}>
+            <a
```

Comment:
We should use Link for internal navigations. See https://nextjs.org/docs/api-reference/next/link